src/sentry/sentry_metrics/metrics_interface.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC, abstractmethod
+from typing import Mapping, Sequence, Union
+
+from sentry.sentry_metrics.use_case_id_registry import UseCaseID
+
+
+class GenericMetricsBackend(ABC):
```

Comment:
Separate from Filippo's comment, I think on the base class we should have docstrings for each method.